src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -186,16 +172,7 @@ private void extractChecksToProprietaryFormat(
      * The KualiCo superclass declares this method as private, so we have to define our own instead of overriding.
      */
     private boolean shouldUseIso20022Format() {
-        if (isIso20022FormatParameterEnabled(CUPdpParameterConstants.CU_USE_ISO20022_FORMAT_IND)) {
-            if (isIso20022FormatParameterEnabled(PdpConstants.ISO20022_FORMAT_IND)) {
-                return true;
-            } else {
-                throw new IllegalStateException("ISO20022_FORMAT_IND cannot be disabled "
-                        + "if CU_USE_ISO20022_FORMAT_IND is enabled");
-            }
-        } else {
-            return false;
-        }
+        return isIso20022FormatParameterEnabled(PdpConstants.ISO20022_FORMAT_IND);
     }
 
     private boolean isIso20022FormatParameterEnabled(final String parameterName) {
```

Comment:
Once CU_ISO20022_FORCE_CREATE_LEGACY_CHECK_FILES is removed it will probably make sense to:
1)  remove method `isIso20022FormatParameterEnabled`
2)  replace the contents of `shouldUseIso20022Format` to be the same as base code

#2 could be done ahead of time, but I am fine doing it later though.

Reply:
Good point @ajd299 , I agree.

Reply:
I updated https://jira.cornell.edu/browse/KFSPTS-32809 with these notes, thanks